docs/sources/awssns.md

```diff
@@ -1,126 +1,105 @@
+            ],
+            "Resource": "arn:aws:sns:*:123456789012:e2e-*"
+        },
+        {
+            "Sid": "VisualEditor1",
+            "Effect": "Allow",
+            "Action": "sns:Unsubscribe",
+            "Resource": "*"
+        }
+    ]
+}
+```
 
-Copy the sample manifest from `config/samples/awssns-sinkbinding.yaml` and replace the pre-filled environment variables
-under `env` with the values corresponding to your _AWS SNS_ topic. Then, create the `Deployment` and `SinkBinding`
-objects in your Kubernetes cluster:
+![Creating an IAM user](../images/awssns-source/sns-user-policy.png)
 
-```console
-$ kubectl -n <my_namespace> create -f my-awssns-sinkbinding.yaml
-```
+## Deploying an Instance of the Source
 
-## Running locally
+Open the Bridge creation screen and add a source of type `AWS SNS`.
 
-Running the event source on your local machine can be convenient for development purposes.
+![Adding an AWS SNS source](../images/awssns-source/bridge-form-sns-source.png)
 
-### In the shell
+In the Source creation form, give a name to the event source and add the following information:
 
-Ensure the following environment variables are exported to your current shell's environment:
+* [**Secret**][accesskey]: Reference to a [TriggerMesh secret][tm-secret] containing an Access Key ID and a Secret Access Key to communicate with the AWS SNS API, as described in the previous sections.
+* [**AWS ARN**][arn]: ARN of the SNS topic, as described in the previous sections.
 
-```sh
-export ARN=<arn_of_my_sns_topic>
-export AWS_ACCESS_KEY_ID=<my_key_id>
-export AWS_SECRET_ACCESS_KEY=<my_secret_key>
-export NAME=my-awssnssource
-export NAMESPACE=default
-export K_LOGGING_CONFIG=''
-export K_METRICS_CONFIG='{"domain":"triggermesh.io/sources", "component":"awssnssource", "configMap":{}}'
-```
+![AWS SNS source form](../images/awssns-source/bridge-form-sns-source-form.png)
 
-Then, run the event source with:
+After clicking the `Save` button, you will be taken back to the Bridge editor. Proceed to adding the remaining components to the Bridge, then submit it.
 
-```console
-$ go run ./cmd/awssnssource
-```
+![Bridge overview](../images/awssns-source/bridge-form-target.png)
 
-### In a Docker container
+A ready status on the main _Bridges_ page indicates that the event source is ready to receive messages from the AWS SNS topic.
 
-Using one of TriggerMesh's release images:
+![Bridge status](../images/awssns-source/bridge-deployed.png)
 
-```console
-$ docker run --rm \
-  -e ARN=<arn_of_my_sns_topic> \
-  -e AWS_ACCESS_KEY_ID=<my_key_id> \
-  -e AWS_SECRET_ACCESS_KEY=<my_secret_key> \
-  -e NAME=my-awssnssource \
-  -e NAMESPACE=default \
-  -e K_LOGGING_CONFIG='' \
-  -e K_METRICS_CONFIG='{"domain":"triggermesh.io/sources", "component":"awssnssource", "configMap":{}}' \
-  gcr.io/triggermesh/awssnssource:latest
-```
+## Event Types
+
+The AWS SNS event source emits events of the following type:
+
+* `com.amazon.sns.notification`
```

Comment:
then the actual SNS message is in the `data` section of the ce

Reply:
I think we can assume knowledge about the CloudEvent spec in all docs?

Reply:
yea i would also assume the reader to be familiar with the CloudEvent spec